src/components/Button/BaseButtonAddPost.js

```diff
@@ -62,7 +62,7 @@ const BaseButtonAddPost = ({
     <MemoIc_pencil
       width={dimen.normalizeDimen(21)}
       height={dimen.normalizeDimen(21)}
-      color={COLORS.white2}
+      color={COLORS.white}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `MemoIc_pencil` component has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the visibility and contrast of the component against its background, especially considering accessibility guidelines.
```diff
-       color={COLORS.white2}
+       color={COLORS.white}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->